anheumai-dparys/concurrency/concurrency.go

```diff
@@ -0,0 +1,26 @@
+package concurrency
+
+type WebsiteChecker func(string) bool
+type result struct {
+	string
+	bool
+}
+
+func CheckWebsites(wc WebsiteChecker, urls []string) map[string]bool {
+	results := make(map[string]bool)
+
+	rch := make(chan result)
```

Comment:
Please try to avoid shortenings of descriptive variables like resultChannel to rch. I mostly go by: If a name requires a comment, then it should have a more descriptive name.

Reply:
Like already written in the other PR comment, I'll adjust variable naming to what is written in most go libs I just haven't had a look so far. From a natural feel I would say that the code is good readable as you easily see that `wc` is a `WebsiteChecker` and it is just used in this context here.